docs/getting_started/tutorial.md

```diff
@@ -331,12 +347,12 @@ Create a template at `blog/templates/blog/blog_page.html`:
 Note the use of Wagtail's built-in `get_parent()` method to obtain the
 URL of the blog this post is a part of.
 
-Now create a few blog posts as children of `BlogIndexPage`.
+Now create a few blog posts as children of "Our Blog" page.
```

Comment:
We should be careful here, the model name is intentional here as we are learning how Wagtail works, the page's title set by the editor is not super relevant.

```suggestion
Now create a few blog posts as children of `BlogIndexPage` (with the title "Our Blog").
```

Something like this could help, it is important that developers learn that there are models for each page type and that each page is an instance of that model. The instance has the title - which is the confusing thing here - maybe we can better word this without missing out on the code structure detail.

Reply:
@lb- For this, I had to revert back to the original content in the wagtail/main branch as I think the illustration added after it is sufficient explanation.